api/package.json

```diff
@@ -19,6 +19,7 @@
     "@types/cors": "^2.8.4",
     "@types/express": "^4.16.1",
     "@types/node-fetch": "^2.1.6",
-    "ts-node": "^8.0.2"
+    "ts-node": "^8.0.2",
+    "typescript": "^3.3.3"
```

Comment:
typescript is already present in top level `package.json`.

Reply:
please remove typescript from here. use top-level `yarn start:api` to run api only